recipes/xsar/meta.yaml

```
@@ -0,0 +1,93 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "xsar" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/umr-lops/xsar.git
+  git_rev: v{{ version }}
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  # sha256: 2b3a0c466fb4a1014ea131c2b8ea7c519f9278eba73d6fcb361b7bdb4fd494e9
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # noarch: generic
+  script: python -m pip install .
+
+requirements:
+  host:
+    - python
+    - aiohttp
+    - attrs
+    - cartopy
+    - dask
+    - gdal
+    - geopandas
+    - importlib_resources
+    - jmespath
+    - lxml
+    - more-itertools
+    - pyproj
+    - rasterio
+    - setuptools_scm
+    - xarray
+    - pip
```

It doesn't look like most of this is needed to run `setup.py`. Also we need a python version for noarch:
```suggestion
    - python >=3.6
    - setuptools_scm
    - pip
```

---

I apply your change but we have a problem with setuptool-scm [LookupError: setuptools-scm was unable to detect version for](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=408381&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=11146)

I don't know what  setuptools_scm want for working, have you an idea ?